syz-manager/html.go

```diff
@@ -103,7 +115,11 @@ func (mgr *Manager) collectStats() []UIStat {
 	defer mgr.mu.Unlock()
 
 	rawStats := mgr.stats.all()
+	repo, _ := vcs.NewRepo(mgr.cfg.TargetOS, mgr.cfg.Type, ".")
```

Comment:
I think it's a wrong way to obtain git revision. The git checkout is not necessary in '.'. We have git revision in the `sys.GitRevision` variable. But it may contain +.

Reply:
Changed to sys.GitRevision, and I didn't see +.